tests/AnnotationTest.php

```diff
@@ -737,27 +737,27 @@ function foo() : array {
                     $a1->bar();',
             ],
             'spaceInType' => [
-                '<?php
+                'code' => '<?php
                     /** @return string | null */
                     function foo(string $s = null) {
                         return $s;
                     }',
             ],
             'missingReturnTypeWithBadDocblockIgnoreBoth' => [
-                '<?php
+                'code' => '<?php
                     /**
                      * @return [bad]
                      */
                     function fooBar() {
                     }',
-                [],
-                [
+                'assertions' => [],
+                'ignored_issues' => [
                     'InvalidDocblock' => Config::REPORT_INFO,
```

Comment:
What does having a report option does here?

Reply:
From what I recall it was an edge-case where this slight suppression was necessary to trigger a bug